react-accordion-component/components/accordion-component.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+
+function Content(props) {
+  return (
+      <div className="ac-content">
+        <p>{ props.content }</p>
+      </div>
+  );
+}
+
+export default class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      currentID: null
+    };
+  }
+
+  handleClick(id) {
+    const targetID = id;
+    const { currentID } = this.state;
+    currentID === targetID
+      ? this.setState({ currentID: null })
+      : this.setState({ currentID: targetID });
+  }
+
+  render() {
+    const { currentID } = this.state;
+    const { data } = this.props;
+    const renderData = data.map(index => (
+        <div key={index.id}>
+          <div className='ac-title' onClick={() => this.handleClick(index.id) }>
+            <h3>{ index.title }</h3>
+          </div>
+          {
+          currentID === index.id
+            ? <Content content={index.content} />
+            : currentID !== index.id
```

Comment:
There is never a need to check the opposite of what was already checked with an `if` statement, that is what the `else` already does. So here you can do:
```JSX
currentID === index.id
  ? <Content content={index.content} />
  : null
```